docs/feature_unicode.md

```diff
@@ -1,56 +1,78 @@
 # Unicode Support
 
-There are three Unicode keymap definition methods available in QMK:
+Unicode characters can be input straight from your keyboard! There are some limitations, however.
 
-## `UNICODE_ENABLE`
+There are three different methods available in QMK:
 
-Supports Unicode up to `0x7FFF`. This covers characters for most modern languages, as well as symbols, but it doesn't cover emoji. The keycode function is `UC(c)` in the keymap, where _c_ is the code point's number (preferably hexadecimal, up to 4 digits long). For example: `UC(0x45B)`, `UC(0x30C4)`.
+## Basic Unicode
 
-## `UNICODEMAP_ENABLE`
+This method supports Unicode code points up to `0x7FFF`. This covers characters for most modern languages, as well as symbols, but it doesn't cover emoji.
 
-Supports Unicode up to `0x10FFFF` (all possible code points). You need to maintain a separate mapping table `const uint32_t PROGMEM unicode_map[] = {...}` in your keymap file. The keycode function is `X(i)`, where _i_ is an array index into the mapping table. The table may contain at most 16384 entries.
+Add the following to your `rules.mk`:
 
-You may want to have an enum to make referencing easier. So, you could add something like this to your keymap file:
+```make
+UNICODE_ENABLE = yes
+```
+
+Then add your `UC(c)` keycodes to your keymap, where _c_ is the code point (preferably in hexadecimal, up to 4 digits long). For example: `UC(0x45B)`, `UC(0x30C4)`.
+
+## Unicode Map
+
+This method supports all possible code points (up to `0x10FFFF`), however, you need to maintain a separate mapping table in your keymap file, which may contain at most 16384 entries.
```

Comment:
```suggestion
This method supports all possible code points (up to `0x10FFFF`); however, you need to maintain a separate mapping table in your keymap file. The table may contain at most 16384 entries.
```
I'd keep the max entries sentence separate.

Reply:
I changed this sentence because I felt that "This method supports all possible code points" might at first suggest you can map every single one of them. Thus the "however" is clarifying that you must use the mapping table *and* that it is limited in size.

Reply:
Oh, that makes sense. Alright, just change to a semicolon then. 😄
```suggestion
This method supports all possible code points (up to `0x10FFFF`); however, you need to maintain a separate mapping table in your keymap file, which may contain at most 16384 entries.
```